src/ontology/mondo-ingest.Makefile

```diff
@@ -369,3 +368,36 @@ slurp-%:
 	$(MAKE) slurp/$*.tsv
 
 slurp-all: slurp-omim slurp-doid slurp-ordo slurp-icd10cm slurp-icd10who slurp-ncit
+
+#############################
+###### Synchronization ######
+#############################
+sync/:
+	mkdir -p $@
+
+# TODO: @Nico
+tmp/mondo-ingest.db:
+	touch tmp/mondo-ingest.db
+tmp/mondo-ingest-without-excluded.db: tmp/mondo-ingest.db
+	cp $< $@
+
+# todo: `pip install` stuff is temporarily here until we come up with a fix. otherwise docker won't work
+sync/mondo-sync-%.owl: tmp/mondo.owl tmp/mondo-ingest-without-excluded.db $(TMPDIR)/mondo.sssom.tsv $(COMPONENTSDIR)/%.owl | sync/
+	pip install --upgrade -r $(RELEASEDIR)/requirements-unlocked.txt
+	python3 $(SCRIPTSDIR)/sync.py \
+	--mondo-path tmp/mondo.owl \
+	--mondo-sans-exclusions-path tmp/mondo-ingest-without-excluded.db \
+    --sssom-map-path $(TMPDIR)/mondo.sssom.tsv \
+	--ontology-path $(COMPONENTSDIR)/$*.owl \
+	--onto-config-path metadata/$*.yml \
+	--outpath-synonyms sync/synonyms-$*.robot.template.tsv \
+	--outpath $@
+
+# TODO: merge all `synonyms-%.robot.template.tsv`. Look at $(REPORTDIR)/excluded_terms_in_mondo_xrefs.tsv
+synonyms.robot.template.tsv:
```

Comment:
I'll use previous `awk` code I wrote to merge `synonyms-%.robot.template.tsv` files.

Reply:
Nico has [just confirmed](https://docs.google.com/document/d/1xdSROYkk16qoQY0jGQap6ihNouG04OLH_jE3uAMpU20/edit?disco=AAABQVD4jrA) that we do indeed want it this way in the synonym sync as well.

Nico wrote:
> Resource is OMIM, ORDO etc. I would personally suggest 1 merged version (all resources included, files split only by "case"), and a resource by resource version (only the "omim confirmed" synonyms). This will make it possible to more quickly merge synonyms from one resource if another poses problems. For example, if NCIT contains many wrong synonyms, if we only have one file, we can not merge it until all of these are manually resolved).